### PR TITLE
[OPS-8820] remove NewRelic

### PR DESCRIPTION
**Pull request by teodorescuserban**

### CODE REVIEW

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

File: src/server.ts [hunk 1]
```diff
@@ -16,16 +16,6 @@ import v4Models from '@unocha/hpc-api-core/src/db';
 import { getTokenFromRequest } from './common-libs/auth';
 import { initializeLogging } from './common-libs/logging';
 
-if (process.env.NEW_RELIC_LICENSE_KEY) {
```
Comment (Pl217) [inline src/server.ts:19]: `NEW_RELIC_LICENSE_KEY` is appearing in two `docker-compose.yml` files. You should remove it from there too.

**Review by Pl217 — CHANGES_REQUESTED** [review]
I don't yet know whether we want/need to remove NewRelic from APIs as well. Waiting for an answer on [this Jira comment](https://humanitarian.atlassian.net/browse/OPS-8820?focusedCommentId=149446).

**Review by unocha-hpc — APPROVED** [review]
Checks have passed and this pull request is ready for manual review

**Review by Pl217 — APPROVED** [review]

**Pull request merged**